cmd/gateway/b2/gateway-b2.go

```diff
@@ -0,0 +1,850 @@
+/*
+ * Minio Cloud Storage, (C) 2017, 2018 Minio, Inc.
```

Comment:
We should add 2019 here.

Reply:
This is a revert not changing anything so will not add 2019